blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/CalendarClock.java

```diff
@@ -0,0 +1,21 @@
+package com.rackspacecloud.blueflood.utils;
+
+import org.joda.time.Instant;
+
+import java.util.Calendar;
+
+public class CalendarClock implements Clock {
```

Comment:
@ChandraAddala @shintasmith I agree that it's probably close enough. My only concern is that they can be configured differently:

1. `new Instant()` calls `DateTimeUtils.currentTimeMillis()` which calls `MillisProvider.getMillis()` on the `cMillisProvider` field. `cMillisProvider` in turn can be changed by any piece of code that calls the various setter methods, chiefly `DateTimeUtils.setCurrentMillisProvider`.
2. `Calendar.getTimeInMillis()` calls `updateTime()` which calls `computeTime` on the class that extends `Calendar`. The `Calendar.getInstance()` call gets a `Calendar` instance via a big complex process, which can be customize... _somewhere_. I don't know where; trying to find it exhausted my patience.

Anyways, it is possible that some part of the codebase, somewhere, would try to change the way time values are calculated via either route. That would affect how the system operates at run-time, and may break something. I think that is very unlikely, however. **If y'all are ok with that, then it's fine with me; I'll change it.**